.env.local.example

```
@@ -117,6 +117,13 @@ E2B_TEMPLATE=terminal-agent-sandbox
 # STRIPE_API_KEY=sk_test_
 # STRIPE_EXTRA_USAGE_WEBHOOK_SECRET=
 
+# =============================================================================
+# TRIGGER.DEV (Required for Agent streaming - non-temporary chats)
+# =============================================================================
+# Get your secret key and project ID at: https://cloud.trigger.dev/
+TRIGGER_SECRET_KEY=tr_dev_...
+TRIGGER_PROJECT=proj_xxxxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRIGGER_PROJECT key should go before the TRIGGER_SECRET_KEY key

(UnorderedKey)